Zend/tests/use_statement/aliasing_builtin_types.phpt

```diff
@@ -0,0 +1,78 @@
+--TEST--
+Aliasing built-in types
+--FILE--
+<?php
+
+/* Taken from zend_compile.c reserved_class_names[] struct */
+/* array and list are also added as they are language constructs */
+$types = [
+    "bool",
+    "false",
+    "float",
+    "int",
+    "null",
+    "parent",
+    "self",
+    "static",
+    "string",
+    "true",
+    "void",
+    "never",
+    "iterable",
+    "object",
+    "mixed",
+    "array",
+    "list",
+];
+
+foreach ($types as $type) {
+    try {
+        eval("use $type as A;");
+    } catch (\Throwable $e) {
+        echo $e::class, ': ', $e->getMessage(), PHP_EOL;
+    }
+    try {
+        eval("use $type as A; function foo$type(A \$v): A { return \$v; }");
```

Comment:
Kind of useless as all the above tests already Compile or ParseError.

Reply:
I added this because if I didn't add the early return in the C code when the use statement is created, it would create a non catchable fatal error.
But yes it is redundant :)